src/1b_prep_aca.jl

```diff
@@ -96,4 +96,61 @@ region_features[!, :geometry] = Vector{AG.IGeometry}(AG.forceto.(region_features
         src_aca_bathy = nothing
         GC.gc()
     end
+
+    if !isfile(joinpath(ACA_OUTPUT_DIR, "$(reg)_waves_Hs.tif"))
+        target_waves_Hs_path = first(glob("*.nc", joinpath(WAVE_DATA_DIR, "Hs", reg)))
+        target_waves_Hs = Raster(target_waves_Hs_path, key=:Hs90, crs=crs(src_aca_bathy), mappedcrs=EPSG(4326), lazy=true)
```

Comment:
I think you will have to define src_aca_bathy after the first if(bathy file) statement, but outside of the turbidity if statement. Because at the moment in the turbidity section src_aca_bathy is removed to nothing.